libc/optee/sysconf.c

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <errno.h>
+#include <openenclave/enclave.h>
+
+long sysconf(int name)
+{
+    OE_UNUSED(name);
+    return 8;
```

Comment:
What does the number "8" mean here?  (add a comment)

Reply:
File removed. This was a workaround from when I ported master's C++ support into our branch.